cilium-health/launch/endpoint.go

```diff
@@ -266,10 +266,15 @@ func LaunchAsEndpoint(owner endpoint.Owner, hostAddressing *models.NodeAddressin
 		return fmt.Errorf("Error while adding endpoint: %s", err)
 	}
 
+	if err := ep.LockAlive(); err != nil {
```

Comment:
Please add a `Fixes: ` tag to the commit message

Reply:
Done and posted as a separate PR. I'd still like this backported to v1.4 because I don't want to break the Istio integration again.